database/migrations/sqls/20240715163245-initial-db-up.sql

```diff
@@ -0,0 +1,86 @@
+DROP TABLE IF EXISTS users cascade;
+
+
+DROP TABLE IF EXISTS admins;
+
+
+DROP TABLE IF EXISTS bidder cascade;
+
+
+DROP TABLE IF EXISTS skill cascade;
+
+
+DROP TABLE IF EXISTS contractor cascade;
+
+
+DROP TABLE IF EXISTS bids cascade;
+
+
+DROP TABLE IF EXISTS tender cascade;
+
+
+DROP TABLE IF EXISTS tender_skill cascade;
+
+
+DROP TABLE IF EXISTS bidder_skill cascade;
```

Comment:
Don't drop tables. If the tables exist it means we did something wrong and we should investigate

Reply:
Alright